lib/metadataTypes/definitions/MobileKeyword.definition.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
     bodyIteratorField: 'entry',
     dependencies: ['mobileCode'],
     hasExtended: false,
-    idField: 'decodedId',
+    idField: 'id',
```

Comment:
i put it with decoded because it is used in other references - not sure if you triple checked this?

Reply:
if you can tell me WHERE then sure thing. in my tests, `id` was used in journey

Reply:
just somewhere in memory is all. If you are ok, then lets move it forward iwth ID